CPT331.Data.Migration/Program.cs

```diff
@@ -37,6 +37,20 @@ static Program()
 
 		private static readonly Options _options;
 
+		private static void CheckGeography(SqlConnection sqlConnection)
+		{
+			OutputStreams.WriteLine($"Checking geography...");
+
+			StringBuilder stringBuilder = new StringBuilder();
+
+			//	Automatically correct any geography nasties - this could cause some geography to shift slightly
+			stringBuilder.AppendLine("UPDATE Location.LocalGovernmentArea SET Area = Area.MakeValid() WHERE Area.STIsValid() = 0;");
```

Comment:
Looks like a hack, but is actually required to fix any geography that might be weird.